src/components/autocomplete/js/autocompleteController.js

```diff
@@ -207,6 +207,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     elements.$    = getAngularElements(elements);
   }
 
+	/**
+	 * Finds the correct element that will determine the width
```

Comment:
The `warp` is used to calculate the width of the menu. I"ll update the description accordingly.